main.nf

```diff
@@ -450,19 +515,40 @@ workflow NOVELTREE {
         .join(ch_all_og_clean_msas)
         .combine(ch_speciesrax)
 
-    GENERAX_PER_FAMILY(
-        ch_generax_input
-    )
-        .generax_per_fam_gfts
-        .collect { it[1] }
-        .set { ch_recon_perfam_gene_trees }
+    // GENERAX_PER_FAMILY: full mode only
+    if (params.workflow_mode == 'full') {
+        GENERAX_PER_FAMILY(
+            ch_generax_input
+        )
+            .generax_per_fam_gfts
+            .collect { it[1] }
+            .set { ch_recon_perfam_gene_trees }
+    }
 
+    // GENERAX_PER_SPECIES: both modes
     GENERAX_PER_SPECIES(
         ch_generax_input
     )
-        .generax_per_spp_gfts
-        .collect { it[1] }
-        .set { ch_recon_perspp_gene_trees }
+
+    ch_recon_perspp_gene_trees = GENERAX_PER_SPECIES.out.generax_per_spp_gfts.collect { it[1] }
+
+    // PHYLO_PROFILES: simplified mode only
+    if (params.workflow_mode == 'simplified') {
+        event_counts_ch = GENERAX_PER_SPECIES.out.event_counts.collect { it[1] }
+        species_event_counts_ch = GENERAX_PER_SPECIES.out.species_event_counts.collect { it[1] }
+        transfer_event_counts_ch = GENERAX_PER_SPECIES.out.transfer_event_counts.collect { it[1] }
+        species_coverage_ch = GENERAX_PER_SPECIES.out.species_coverage.collect { it[1] }
+        ogs_ch = GENERAX_PER_SPECIES.out.event_counts.collect { it[0].og }
+
+        PHYLO_PROFILES(
+            event_counts_ch,
+            species_event_counts_ch,
+            transfer_event_counts_ch,
+            species_coverage_ch,
+            ogs_ch,
+            ORTHOFINDER_MCL_ALL.out.inflation_dir
+        )
+    }
```

Comment:
Honestly we probably always want to include the phylo profiles analyses into any/all modes, as these really is just reformatting of these results into more useful tabular formats. So, I think we can probably just take this out of the simplified mode.